oauth2_http/javatests/com/google/auth/oauth2/ComputeEngineCredentialsTest.java

```diff
@@ -190,6 +190,98 @@ public void buildTokenUrlWithScopes_defaultScopes() {
     assertEquals("bar", scopes.toArray()[1]);
   }
 
+  @Test
+  public void buildTokenUrl_nullTransport() {
```

Comment:
nit: one additional test where both transport and bindingenforcements are null

Reply:
Done in https://github.com/googleapis/google-auth-library-java/pull/1558/commits/ce74e45f2cb2fba09486fe4faccac933950ebd09